src/sentry/utils/appleconnect/appstore_connect.py

```diff
@@ -192,7 +195,7 @@ def get_apps(session: Session, credentials: AppConnectCredentials) -> Optional[L
     try:
         app_pages = _get_appstore_info_paged(session, credentials, url)
         for app_page in app_pages:
-            for app in app_page["data"]:
+            for app in safe.get_path(app_page, "data", default=[]):
```

Comment:
i kind of liked the sentry error that would occur if `data` wasn't present.  i doubt that processing further apps would help much here over having the whole task bail out.  I'm in general rather conflicted on all this try and progress the task gracefully as much as we can that's happening to be honest.  Come to think of it the same applies to the ones above.  I'm happy to accept this as-is, but hmm, maybe i'll attempt a followup.